src/main/java/domain/user/User.java

```diff
@@ -0,0 +1,69 @@
+package domain.user;
+
+import domain.ladder.LadderBridge;
+import domain.user.exception.InvalidNameException;
+import domain.user.exception.InvalidNameLengthException;
+
+import java.util.List;
+import java.util.Objects;
+
+public class User {
+    public static final String ALL = "all";
+    private static final int NAME_LENGTH_CONDITION = 5;
+
+    private final String name;
+    private final int position;
+
+    public User(final String name, final int position) {
+        validName(name);
+        validNameLength(name);
+        this.name = name;
+        this.position = position;
+    }
+
+    private void validName(final String name) {
+        if (ALL.equals(name)) {
+            throw new InvalidNameException();
+        }
+    }
+
+    private void validNameLength(final String name) {
+        if (name.length() > NAME_LENGTH_CONDITION) {
+            throw new InvalidNameLengthException();
+        }
+    }
+
+    public User move(final List<LadderBridge> ladderBridges) {
+        if (ladderBridges.size() == position) {
```

Comment:
LadderLine 클래스가 List<LadderBridge>를 관리하는 것 같은데 ladderline 객체에 메세지를 보내서 마지막 위치인지 확인하는게 어떤가요?

```suggestion
	public User move(final LadderLine ladderline) {
        if (ladderline.isLastPosition(position)) {
```
만약 구현하게 된다면 위와 같은 모습이 될 거 같네요. :) move를 사용하고 있는 부분을 타고 넘어갔을 때 LadderGame의 move가 나오는데 여기서 `ladderline.getLadderBridges()`를 부르고 있는걸로 봐서 충분히 위 방식으로 구현할 수 있을거라 생각합니다!